### DISCUSSION TITLE
Custom 404 page via vercel.json not showing?

### DISCUSSION THREAD

**Discussion by drbirkin** [vercel/community#671] [Help] [answered]
HI I followed the documentation  https://vercel.com/guides/custom-404-page

`{
  "routes": [
    { "handle": "filesystem" },
    { "src": "/(.*)", "status": 404, "dest": "/404.html" }
  ]
}
`
and after deployment it wont redirect me to my custom 404 page which is 404.html
<img width="113" alt="image" src="https://user-images.githubusercontent.com/57367079/174496574-d2322a41-f51c-44e6-892a-e97cb8f8517f.png">

<img width="389" alt="image" src="https://user-images.githubusercontent.com/57367079/174496595-810669b7-0952-426a-a2a5-7072dd87f0cc.png">

What do I miss, Im not using any frameworks. Many thanks!

**Comment by swarnava** [comment, marked as answer]
Please create a case here https://vercel.com/support/request so that we can take a closer look.
